op-batcher/batcher/espresso.go

```diff
@@ -341,12 +323,15 @@ func evaluateVerification(jobResp espressoVerifyReceiptJobResponse) JobEvaluatio
 		return Handle
 	}
 
-	// If the hash is invalid, skip the verification.
-	if strings.Contains(err.Error(), "hash is nil") {
-		log.Warn("Hash is nil, skipping")
+	if errors.Is(err, espressoClient.ErrPermanent) {
 		return Skip
 	}
 
+	if !errors.Is(err, espressoClient.ErrEphemeral) {
```

Comment:
Does this mean that in this case the error is not permanent nor ephemeral? Is this kind of an unexpected error then?